CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - new static API : Sentry.init(), Sentry.captureEvent() #108
 - expect a sdkName based on the test platform #105
 - Added Scope and Breadcrumb ring buffer #109
+- Added Hub ring buffer #113
```

Comment:
😳

Reply:
Sorry if twas confused. Maybe we could add this entry as:

"Added Hub to SDK" or "added Hub"? 

I just think "ring buffer" is confusing since we have that for breadcrumbs but not here.
This PR includes a stack of scopes though maybe that's the confusion?

Reply:
yes sorry :) I was just shocked by what I wrote (copy/pasted :D ) !